packages/yourloops/components/menus/user-menu.tsx

```diff
@@ -0,0 +1,159 @@
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import CancelIcon from "@material-ui/icons/Cancel";
+import ContactSupportOutlinedIcon from "@material-ui/icons/ContactSupportOutlined";
+import FaceIcon from "@material-ui/icons/Face";
+import PermContactCalendarIcon from "@material-ui/icons/PermContactCalendar";
+import StethoscopeIcon from "../icons/StethoscopeIcon";
+
+import { makeStyles } from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import Divider from "@material-ui/core/Divider";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import MenuItem from "@material-ui/core/MenuItem";
+import Typography from "@material-ui/core/Typography";
+
+import { UserRoles } from "../../models/shoreline";
+import { useAuth } from "../../lib/auth";
+import RoundedHospitalIcon from "../icons/RoundedHospitalIcon";
+import config from "../../lib/config";
+import metrics from "../../lib/metrics";
+import MenuLayout from "../layouts/menu-layout";
+
+const classes = makeStyles(() => ({
+  clickableMenu: {
+    cursor: "pointer",
+  },
+  svgIcon: {
+    margin: "inherit",
+  },
+}));
+
+function UserMenu(): JSX.Element {
+  const { t } = useTranslation("yourloops");
+  const { user, logout } = useAuth();
+  const { svgIcon, clickableMenu } = classes();
+  const history = useHistory();
+
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+  const opened = !!anchorEl;
+
+  const getRoleIcon = (): JSX.Element => {
+    switch (user?.role) {
+    case UserRoles.hcp:
+      return <StethoscopeIcon />;
+    case UserRoles.caregiver:
+      return <RoundedHospitalIcon />;
+    default:
+      return <FaceIcon />;
+    }
+  };
+
+  const closeMenu = () => setAnchorEl(null);
+
+  const onClickSettings = () => {
+    history.push("/preferences");
+    closeMenu();
+  };
+
+  const onClickLogout = async () => {
+    await logout();
+    closeMenu();
+  };
+
+  const onClickSupport = () => {
+    window.open(config.SUPPORT_WEB_ADDRESS, "_blank");
+    closeMenu();
+    metrics.send("support", "click_customer_service");
+  };
+
+  return (
+    <React.Fragment>
+      <Box
+        id="user-menu"
+        display="flex"
+        alignItems="center"
+        className={clickableMenu}
+        onClick={event => setAnchorEl(event.currentTarget)}
+      >
+        <Box id="user-role-icon">
+          {getRoleIcon()}
+        </Box>
+        <Box marginX={1}>
+          <Typography>
+            {`${user?.role === UserRoles.hcp ? "Dr" : ""} ${user?.fullName}`}
```

Comment:
Is "Dr" adequate for all languages? Even japanese? Is everyone with the HCP role a doctor?

Reply:
We remove the Dr prefix at the moment ;)